dot/state/block.go

```diff
@@ -280,6 +280,17 @@ func (bs *BlockState) GetBlockBody(hash common.Hash) (*types.Body, error) {
 	return types.NewBody(data), nil
 }
 
+// GetFinalizedHead returns the latest finalized block header
+// TODO: relies on GRANDPA implementation. currently returns genesis header.
```

Comment:
do we have a specific issue we can tag to track this?

Reply:
no, can add one

Reply:
#883